Source/WebKit/WebProcess/GPU/media/RemoteVideoFrameProxy.cpp

```diff
@@ -127,6 +127,14 @@ CVPixelBufferRef RemoteVideoFrameProxy::pixelBuffer() const
         m_pixelBuffer = WebCore::createBlackPixelBuffer(static_cast<size_t>(m_size.width()), static_cast<size_t>(m_size.height()));
     return m_pixelBuffer.get();
 }
+
+Vector<uint8_t> RemoteVideoFrameProxy::resourceIdentifierVector() const
+{
+    auto ref = newReadReference();
+    Vector<uint8_t> result(sizeof(ref));
```

Comment:
Vector<uint8_t> result { reinterpret_cast< uint8_t*>(&ref), sizeof(ref) };

Reply:
I got rid of the memcpy / Vector entirely :)